src/layer/__tests__/WmsLayer.ts

```diff
@@ -0,0 +1,38 @@
+import 'jest-setup';
+
+import { BBox, CRS_EPSG4326, ApiType, MimeTypes, WmsLayer } from 'src';
+
+test('WmsLayer.getMapUrl returns an URL', () => {
+  const bbox = new BBox(CRS_EPSG4326, 19, 20, 20, 21);
+  const layerId = 'PROBAV_S1_TOA_333M';
+  const layer = new WmsLayer(
+    'https://proba-v-mep.esa.int/applications/geo-viewer/app/geoserver/ows',
+    layerId,
+  );
+
+  const getMapParams = {
+    bbox: bbox,
+    fromTime: new Date(Date.UTC(2020, 1 - 1, 10, 0, 0, 0)), // 2020-01-10/2020-01-10
+    toTime: new Date(Date.UTC(2020, 1 - 1, 10, 23, 59, 59)),
+    width: 512,
+    height: 512,
+    format: MimeTypes.JPEG,
+  };
+  const imageUrl = layer.getMapUrl(getMapParams, ApiType.WMS);
+
+  expect(imageUrl).toHaveOrigin('https://proba-v-mep.esa.int');
```

Comment:
Should baseUrl also be tested somewhere?

Reply:
Maybe, though this PR and #22 are mostly about setting up the tools that we will need for testing, not so much about the tests themselves. I did add a test just to show that these tools work, but I would prefer to keep them small and we can complete them in the future, if you agree?

Reply:
OK